DataPickerViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DataPickerViewController.swift
+//  outtatime
+//
+//  Created by Lambda_School_Loaner_268 on 1/29/20.
+//  Copyright © 2020 Jeremiah. All rights reserved.
+//
+
+import UIKit
+
+protocol DatePickerDelegate {
+    func destinationDateWasChosen(date: Date)
+}
+
+class DataPickerViewController: UIViewController {
+    var delegate: DatePickerDelegate?
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+
+        // Do any additional setup after loading the view.
+    }
+    @IBOutlet weak var datePicker: UIDatePicker!
+    
+    @IBAction func Cancel(_ sender: UIBarButtonItem) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func Done(_ sender: UIBarButtonItem) {
+        delegate!.destinationDateWasChosen(date: datePicker.date)
```

Comment:
Make sure to use optional chaining rather than force chaining here.
EX: `delegate?.destinationDateWasChosen(date: datePicker.date)`